ansible/roles/update_node/tasks/main.yml

```diff
@@ -0,0 +1,68 @@
+---
```

Comment:
I have discussed with @dak1n1 about combining this role with the update_master.  Please see the comment page for some thoughts.